internal/gh/client.go

```diff
@@ -32,6 +37,14 @@ func NewClient(token string) (*Client, error) {
 	return &Client{httpClient, githubv4.NewClient(httpClient)}, nil
 }
 
+func GetClient(token string) (*Client, error) {
```

Comment:
@travigd I think this way of lazy initializing the GH client is a better pattern, but let me know if I missed a super obvious reason why this wouldn't work.